src/api/.rubocop.yml

```diff
@@ -55,3 +55,7 @@ Style/AndOr:
 
 Style/EmptyElse:
   EnforcedStyle: both
+
+Style/HashSyntax:
+  EnforcedStyle: ruby19
```

Comment:
\*Updated\*

As written in the second commit: "This means that in cases where we have a stringified key, the whole hash
needs to be in hash rocket syntax."